ui-events/src/pointer/mod.rs

```diff
@@ -214,6 +214,28 @@ impl PointerUpdate {
     }
 }
 
+/// An event representing a [`PointerButton`] that was pressed or released.
+#[derive(Clone, Debug)]
+pub struct PointerButtonPressEvent {
```

Comment:
I think the word `Press` here is both redundant and misleading, so `PointerButtonEvent` is plenty.

Reply:
I thought this name was too long, but I couldn't work out which bit to remove!